src/CheckableAvatar.js

```diff
@@ -32,7 +32,7 @@ const styleSheet = () => (
  *
  * CheckableAvatar is a controlled component so the state needs to be managed. This is to allow more fine grained control.
  */
-const CheckableAvatar = ({ isCheckable, image, onClick, ...rest }) => {
+const CheckableAvatar = ({ isCheckable, children, onClick, ...rest }) => {
```

Comment:
why are you passing children as a prop? Looks like a redundant interface, to what react provides via `this.children`

Reply:
Children is on props so destructuring for clarity.

`<Thing children="Hello"/> === <Thing>Hello</Thing>`

Reply:
You're right